doc/source/getting-started-mac-tutorial.rst

```diff
@@ -0,0 +1,155 @@
+macOS Tutorial
+==============
+
+This guide will give you a brief overview of how volatility3 works as well as a demonstration of several of the plugins available in the suite.
+
+Acquiring memory
+----------------
+
+Volatility3 does not provide the ability to acquire memory. The example below is an open source tool. Other commercial tools are also available.
+
+* `osxpmem <https://github.com/Velocidex/c-aff4/releases/download/3.2/osxpmem_3.2.zip>`_
+
+
+
+Procedure to create symbol tables for macOS
+--------------------------------------------
+
+To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.
+
+.. tip:: It may be possible to locate pre-made ISF files from the `download link <https://downloads.volatilityfoundation.org/volatility3/symbols/mac.zip>`_ ,
+    which is built and maintained by `volatilityfoundation <https://www.volatilityfoundation.org/>`_.
+    After creating the file or downloading it from the link, place the file under the directory ``volatility3/symbols/mac``.
+    If necessary create a mac directory under the symbols directory (this will become unnecessary in future versions).
```

Comment:
I think it's already no longer necessary, so for any version of the code this documentation would be attached to, this line should be unnecessary.  Since we've moved to a faster release cycle, the database work has already been done and the requirement on os directories for the ISF files should no longer be there.  Sorry for the change in guidance, but luckily it's because we're still improving things...  5;)

Reply:
All right. I've updated the directory to point to "volatility3/symbols/" and removed any information about creating subdirectories.